arena_bringup/configs/benchmark/suites/meta_suite.yaml

```diff
@@ -0,0 +1,384 @@
+stages:
+  - name: map_empty_1_jackal
+    episodes: 5
```

Comment:
Does each scenario need to be run 5 times?

Reply:
yes, this is how we benchmarked it for our paper

Reply:
we could also change it to 1 since you can scale it in the config, but it's better like it is right now I think